bin/submarine-daemon.sh

```diff
@@ -39,7 +39,6 @@ SUBMARINE_SERVER_MAIN=org.apache.submarine.server.SubmarineServer
 JAVA_OPTS+="${SUBMARINE_SERVER_JAVA_OPTS} ${SUBMARINE_SERVER_MEM} -Dsubmarine.log.file=${SUBMARINE_SERVER_LOGFILE}"
 
 add_jar_in_dir "${BIN}/../lib"
-add_jar_in_dir "${BIN}/../lib/submitter"
```

Comment:
This is used to fix jar conflicts?

Reply:
No, this is an invalid code.
Because the submitter is loaded dynamically at runtime.